packages/vb-decompiler-lite.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,17 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try{
+
+    $toolName = 'VB Decompiler Lite'
+    $category = 'Visual Basic'
+
+    $zipUrl = 'https://www.vb-decompiler.org/files/vb_decompiler_lite.zip'
```

Comment:
This URLs does not include the version and will break with an update as the hash will change. Any chance we can get a URL that includes the version?

Reply:
unfortunately not, the version is only included in the body of the html but not in the URL

Reply:
only included in older versions, for example: https://www.vb-decompiler.org/files/vb_decompiler_lite_84.zip

Reply:
how old are the older versions? If they are not too old, I prefer this option instead of having a URL that will eventually break. We try not to add packages to the default config for which we know the hash will break, as this confuses FLARE-VM users.

Reply:
@sara-rn :eyes:

Reply:
It seems current version is 12.4 and the one in the link is 8.4, so it is quite old. According to https://www.vb-decompiler.org/news.htm, it seems VB decompiler releases a new version 1-3 times per year. I think it is ok to use a hash that may break for this release cadence. I propose merging this Pr as it is and considering improving this package at a later point.